### PR TITLE
146 us enhance use case 2 functionalities

### PR DESCRIPTION
**Pull request by witnick**
Added stored procedures to use case 2: US #146 

- Get all assigned ticket for a given user
- Add comments to a ticket
- Get all the comments for a given ticket
- Assign a ticket to a given user

To test the function I would suggest postman or the Thunder Client for VS code
1.  Start your containers and then Run command `./refresh_database.sh -f '/useCase2' `
2. Import the json from path ~\UInnovate\Postman Collections\UseCase2.postman_collection.json
3. Once loaded, you should have something that looks like this:
image
From there you can read the table with the GET commands and use the POST commands to update the state of the database.

### CODE REVIEW

File: init.sql
Comment (GuardiansAscend) [inline init.sql:1]: These are all comments right? im not sure this code belongs in init.sql, but we can discuss to be sure

Comment (witnick) [reply init.sql:1]: they aren't run. I removed them

**Review by GuardiansAscend — DISMISSED** [review]
Perfectly well done (just how we like it), approved!

**Review by lyubageorgieva — APPROVED** [review]
Looks Good!

**Pull request merged**